BigQuery/src/QueryResults.php

```diff
@@ -338,7 +339,7 @@ public function isComplete()
      * $job = $queryResults->job();
      * ```
      *
-     * @return Google\Cloud\BigQuery\Job
+     * @return Job
```

Comment:
I noticed there was an attempt to fix this with https://github.com/googleapis/google-cloud-php/pull/6443 but it's missing a `\` at the beginning so it was being interpreted as  `Google\Cloud\BigQuery\Google\Cloud\BigQuery\Job`.